### PR TITLE
Fix Prettier/ESLint for grants and signup

### PR DESCRIPTION
**Pull request by varortz**
[//]: # "These comments are meant for your reference. They are invisible and don't need to be deleted!"

### :cloud: :cloud: Description :cloud: :cloud:
[//]: # "REQUIRED - Describe what's new in this PR in a few lines. A description and bullet points for specifics will suffice."

- Made Prettier/ESlint style changes for the Grants screen and two sign-up screens (SignUp1 and SignUp2).
- Left a `console.error(e)` on SignUp1.tsx (line 63) which was producing a warning with ESlint
- Fixed import statements (order of import statements and removal of unused imports)

### Screenshots :calling:
[//]: # "REQUIRED for frontend changes, otherwise optional but strongly recommended. Add screenshots of expected behavior - GIFs if you're feeling fancy! If you are making changes here, please CC: @leexesther at the bottom."

## :zap: How to review :zap:
[//]: # 'REQUIRED - Describe the order in which to review files and what to expect when testing locally. Is there anything specifically you want feedback on? Should this be reviewed commit by commit, or all at once? What are some user flows to test? What are some edge cases to look out for?'

- Run `npx eslint ./src/screens/GrantsScreen`, `npx eslint ./src/screens/SignUp1Screen`, and `npx eslint ./src/screens/SignUp2Screen` to ensure that there are no more ESlint errors.
- Was able to build the app using `npx expo run:ios`

## :seedling: Next steps :seedling:
[//]: # "Optional - What's NOT in this PR, doesn't work yet, and/or still needs to be done. Note any temporary fixes in this PR that should be cleaned up later."

### Online sources :information_source:
[//]: # 'Optional - copy links to any tutorials or documentation that was useful to you when working on this PR'

### Related PRs :raised_hands:
[//]: # "Optional - related PRs you're waiting on/ PRs that will conflict, etc; if this is a refactor, feel free to add PRs that previously modified this code"

[//]: # 'This tags the project leader as a default. Feel free to change, or add on anyone who you should be in on the conversation.'
CC: @davidqing6432

### CODE REVIEW

File: RealmApp.tsx [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import Realm from "realm";
 
-let mongoId = process.env.REACT_APP_ID;
```
Comment (wangannie) [inline RealmApp.tsx]: Make sure you revert this! We don't wanna commit the actual ID

File: src/screens/GrantsScreen/Grants.tsx [hunk 2]
```diff
@@ -1,45 +1,43 @@
...
-import ViewContainer from "../../components/ViewContainer";
-import RectButton from "../../components/RectButton";
-import globalStyles from "../../globalStyles";
-import styles from "./styles";
+import { Text } from 'react-native';
+import { useTranslation } from 'react-i18next';
+import ViewContainer from '../../components/ViewContainer';
+import RectButton from '../../components/RectButton';
+import globalStyles from '../../globalStyles';
+// import styles from './styles';
```
Comment (wangannie) [inline src/screens/GrantsScreen/Grants.tsx]: Remove comment

File: sync.config.js [hunk 3]
```diff
@@ -1,4 +1,4 @@
-let mongoId = process.env.REACT_APP_ID;
```
Comment (wangannie) [inline sync.config.js]: Revert this!

File: src/screens/SignUp2Screen/SignUp2.tsx [hunk 4]
```diff
@@ -1,18 +1,18 @@
...
+import ViewContainer from '../../components/ViewContainer';
+import RectButton from '../../components/RectButton';
+import { RootStackScreenProps } from '../../types/navigation';
+import globalStyles from '../../globalStyles';
+import styles from './styles';
+import BackButton from '../../components/BackButton';
+import LabeledTextInput from '../../components/LabeledTextInput';
 
-const SignUp2Screen = ({ navigation }: any) => {
+function SignUp2Screen({ navigation }: RootStackScreenProps<'SignUpScreen2'>) {
```
Comment (wangannie) [inline src/screens/SignUp2Screen/SignUp2.tsx]: Should be `SignUp2`

**Review by wangannie — APPROVED** [review]
Approving since I made the changes directly on your branch (so we can unblock other work)
